README.md

```diff
@@ -34,37 +47,89 @@ let package = Package(
 )
 ```
 
-### Configuring the Raygun client
+### With CocoaPods
+
+To integrate Raygun using CocoaPods, update your Podfile to include:
 
-The configuration step is slightly different if installed via the Swift package manager. Rather than importing the specific header for the target application, simply import `raygun4apple`.
+```bash
+pod 'raygun4apple'
+```
 
-Here is an example which imports raygun4apple, initializes the provider, and sends a test exception.
+Once updated, you can run `pod install` from Terminal.
+
+### With GitHub releases
+
+The latest release can be found [here](https://github.com/MindscapeHQ/raygun4apple/releases). The frameworks are attached to each release as a zipped file. This can be downloaded, unzipped and included in you project directory.
+
+Once included, go to your app's target **General** settings and add the raygun4apple framework to the **Frameworks, Libraries, and Embedded Content** section. Ensure that the framework is set to **Embed & Sign**.
+
+## Using Raygun
+
+### Swift
+
+To use Raygun, import the package in your Swift file. Here is an example which imports raygun4apple, initializes the provider, and sends a test exception. Be sure to replace `YOUR_API_KEY_HERE` with your API key from your Application Settings screen in Raygun.
 
 ```swift
 import raygun4apple
 
 let raygunClient = RaygunClient.sharedInstance(apiKey: "YOUR_API_KEY_HERE")
 raygunClient.enableCrashReporting()
-raygunClient.send(exception: NSException.init(name: NSExceptionName.illegalSelectorException, reason: "This is a macOS error!"))
+raygunClient.send(exception: NSException.init(name: .genericException, reason: "This is an exception from Raygun4Apple!"))
 ```
 
-### With CocoaPods
+NB: If you recieve a `A server with the specified hostname could not be found` error, you may need to [enable outbound connections for your app](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_security_network_client). Choose your project target in the left-hand navigator, then choose the "Signing & Capabilities" tab, and check the `Outgoing Connections (Client)` in the "App Sandbox" section.
 
-To integrate Raygun using CocoaPods, update your Podfile to include:
+#### Swift UI
 
-```bash
-pod 'raygun4apple'
+You likely want to start Raygun in your `AppDelegate`. By default newer Swift UI apps do not come with an `AppDelegate`, so you can [follow these instructions](https://www.hackingwithswift.com/quick-start/swiftui/how-to-add-an-appdelegate-to-a-swiftui-app) to add one to your project (for macOS you need to [use NSApplication](https://stackoverflow.com/questions/71291654/swiftui-appdelegate-on-macos) instead)
```

Comment:
When I create a new Swift project, instead of the `NSObject` type I have the `UIResponder` type in that same position. Should I replace that with `NSObject`

Reply:
@sumitramanga I think it depends on what kind of project you're creating - for macOS or for iOS. Can you paste the `App` struct you have in here?

Reply:
```
@main
class AppDelegate: NSObject, UIApplicationDelegate {

    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]? = nil) -> Bool {
        
        let raygunClient = RaygunClient.sharedInstance(apiKey: "OHsrSPwZLyenoubIzoNSA")
        raygunClient.enableCrashReporting()

        return true
    }

    func application(_ application: UIApplication, configurationForConnecting connectingSceneSession: UISceneSession, options: UIScene.ConnectionOptions) -> UISceneConfiguration {
        return UISceneConfiguration(name: "Default Configuration", sessionRole: connectingSceneSession.role)
    }

    func application(_ application: UIApplication, didDiscardSceneSessions sceneSessions: Set<UISceneSession>) {
    }

}```

Reply:
I created a iOS app with Swift This is the AppDelegate that I've been working in. I also have been working in the ViewController which I'll show you. These are the only files I've touched

Reply:
```

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        
        let crashButton = UIButton(type: .system)
        crashButton.setTitle("Trigger crash", for: .normal)
        crashButton.addTarget(self, action: #selector(triggerCrash), for: .touchUpInside)
        
        crashButton.translatesAutoresizingMaskIntoConstraints = false
        view.addSubview(crashButton)
        NSLayoutConstraint.activate([
            crashButton.centerXAnchor.constraint(equalTo: view.centerXAnchor),
            crashButton.centerYAnchor.constraint(equalTo: view.centerYAnchor),
        ])
    }
    
    @objc func triggerCrash() {
        fatalError("Test for Raygun")
    }
}

```

Reply:
@sumitramanga yeah so `NSObject` is just a generic object whereas [`UIResponder` is an interface for events](https://developer.apple.com/documentation/uikit/uiresponder) so I would put `UIResponder` in there. Different people might do different things though. 

Your second example looks like a UIKit app rather than a SwiftUI app, which is why there's a difference. I _think_ they start with an AppDelegate by default.